gulpfile.js

```diff
@@ -40,6 +39,7 @@ var webpack2 = require('webpack');
 var webpackStream = require('webpack-stream');
 var Vinyl = require('vinyl');
 var vfs = require('vinyl-fs');
+var through = require('through2');
```

Comment:
If this patch works it's by luck alone, since you must add the [through2](https://www.npmjs.com/package/through2) package to https://github.com/mozilla/pdf.js/blob/master/package.json for this to be correct.